net-mgmt/zabbix-agent/src/opnsense/service/conf/actions.d/actions_zabbixagent.conf

```diff
@@ -0,0 +1,34 @@
+[setup]
+command:/usr/local/opnsense/scripts/OPNsense/ZabbixAgent/setup.sh
+parameters:
+type:script_output
+
+[start]
+command:/usr/local/opnsense/scripts/OPNsense/ZabbixAgent/setup.sh; /usr/local/etc/rc.d/zabbix_agentd start
+parameters:
+type:script
+message:starting zabbix_agentd
+
+[stop]
+command:/usr/local/etc/rc.d/zabbix_agentd stop; /usr/bin/killall zabbix_agentd; exit 0
+parameters:
+type:script
+message:stopping zabbix_agentd
+
+[restart]
+command:/usr/local/opnsense/scripts/OPNsense/ZabbixAgent/setup.sh; /usr/local/etc/rc.d/zabbix_agentd restart
+parameters:
+type:script
+message:restarting zabbix_agentd
+
+[status]
+command:/usr/local/etc/rc.d/zabbix_agentd status || exit 0
```

Comment:
doesn't this return the status if you let it? not as script_output, then check for error